src/backend/gl/src/native.rs

```diff
@@ -96,6 +96,8 @@ pub struct DescriptorSetLayout;
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
 pub struct DescriptorSet;
 
+#[allow(missing_copy_implementations)]
```

Comment:
I find this one to be the most annoying warning of all. I'd prefer to just put it as a top-level thing. In fact, it **is already**: https://github.com/gfx-rs/gfx/blob/ad34a1eb2590b6fed48c0d8889ef104b490f1eb3/src/backend/gl/src/lib.rs#L4